samples/HubConnectionSample/HubConnectionSample.cpp

```diff
@@ -77,12 +77,12 @@ void chat()
         }
 
         std::cout << "Enter your message:";
-        for (;;)
+        for (; connection.get_connection_state() == signalr::connection_state::connected;)
```

Comment:
What do you have agains while loops?

Reply:
😆 too lazy to change what was already there